src/frontend/components/Felleskomponenter/Pensjonsmodal/Pensjonsperiode.tsx

```diff
@@ -56,7 +57,9 @@ export const Pensjonsperiode: React.FC<Props> = ({
     barn,
 }) => {
     const { erÅpen: pensjonsmodalErÅpen, toggleModal: togglePensjonsmodal } = useModal();
-
+    const pensjonsperiodeSted = gjelderUtlandet
```

Comment:
Samme kommentar som på arbeidsperiode. Litt forvirret av det høres ut som et sted, når man egentlig bare får en ID for en periode